xml/admin_ceph_upgrade.xml

```diff
@@ -931,83 +951,93 @@ mds standby for name = mds.456-another-mds
+   </para>
+   <para>
+    Refer to
+    <link xlink:href="https://documentation.suse.com/suse-distribution-migration-system/1.0/single-html/distribution-migration-system/"/>
+    to find both general and detailed information about DMS.
+   </para>
+   <procedure>
+    <step>
+     <para>
+      Install the migration RPM packages. They adjust the &grub; boot loader to
+      automatically trigger the upgrade on next reboot. Install the
+      <package>SLES15-SES-Migration</package> and
+      <package>suse-migration-sle15-activation</package> packages:
+     </para>
+<screen>&prompt.sminion;zypper install SLES15-SES-Migration suse-migration-sle15-activation</screen>
+    </step>
+    <step>
+     <substeps>
+      <step>
+       <para>
+        If the node being upgraded <emphasis role="bold">is</emphasis>
+        registered with a repository staging system such as SCC, SMT, RMT, or
+        &susemgr;, create the
+        <filename>/etc/sle-migration-service.yml</filename> with the following
+        content:
+       </para>
 <screen>
-&prompt.root;zypper mr -d -a
+use_zypper_migration: true
+preserve:
+  rules:
+    - /etc/udev/rules.d/70-persistent-net.rules
 </screen>
-     </step>
-     <step>
-      <para>
-       Manually add the following software repositories:
-      </para>
-      <itemizedlist>
-       <listitem>
-        <para>
-         SLE-Product-SLES/15-SP1
-        </para>
-       </listitem>
-       <listitem>
-        <para>
-         SLE-Module-Basesystem/15-SP1
-        </para>
-       </listitem>
-       <listitem>
-        <para>
-         SLE-Module-Server-Applications/15-SP1
-        </para>
-       </listitem>
-       <listitem>
-        <para>
-         Storage/6
-        </para>
-       </listitem>
-      </itemizedlist>
-      <para>
-       You can add them by finding out the right repository URL and running:
-      </para>
+      </step>
+      <step>
+       <para>
+        If the node being upgraded is <emphasis role="bold">not</emphasis>
+        registered with a repository staging system such as SCC, SMT, RMT, or
+        &susemgr;, perform the following changes:
+       </para>
+       <substeps>
+        <step>
+         <para>
+          Create the <filename>/etc/sle-migration-service.yml</filename> with
+          the following content:
+         </para>
 <screen>
-zypper ar -f <replaceable>REPO_URL</replaceable>
+use_zypper_migration: false
+preserve:
+  rules:
+    - /etc/udev/rules.d/70-persistent-net.rules
 </screen>
+        </step>
+        <step>
+         <para>
+          Disable or remove the SLE 12 SP3 and SES 5 repos, and add the SLE 15
+          SP1 and SES6 repos. Find the list of related repositories in
+          <xref linkend="upgrade-prepare-repos"/>.
+         </para>
+        </step>
+       </substeps>
+      </step>
+     </substeps>
+    </step>
+    <step>
+     <para>
+      Reboot to start the upgrade. The node will reboot automatically after the
+      upgrade.
```

Comment:
Please add something like "While the upgrade is running, it's possible to ssh in as the migration user, using an existing ssh key from the host system, as described in the DMS documentation. For SES, it is also possible to log in as root on the system console, with the password "sesupgrade" (i.e. if you have physical access to the machine, or direct console access)".